modules/custom/az_global_footer/az_global_footer.install

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/**
+ * @file
+ * Contains az_global_footer.install.
+ */
+
+use Drupal\migrate\MigrateExecutable;
+use Drupal\migrate\MigrateMessage;
+
+/**
+ * Implements hook_install().
+ */
+function az_global_footer_install() {
+  $tag = 'Quickstart Global Footer';
```

Comment:
@danahertzberg @akslay This is all that should be needed to enable that theme setting when the module is installed.  Are we sure that's what we want to do, though?  Would it be better to change the default value for that setting in the theme and add a DB update to change it on existing sites or is it really only needed when this module is installed?

```suggestion
  // Enable clean wrapper for footer_sub region.
  $config = \Drupal::service('config.factory')->getEditable('az_barrio.settings');
  $config
    ->set('bootstrap_barrio_region_clean_footer_sub', TRUE)
    ->save(TRUE);

  $tag = 'Quickstart Global Footer';
```

Reply:
So I believe it's only needed when this module is installed because if it is checked on sites not using it, a block with a bootstrap grid in this region does not display properly.

Reply:
So we we want to automatically disable the setting if the module is uninstalled?

Reply:
We decided against doing this step in our 2022-01-14 Friday meeting.